src/coreclr/inc/gcinfodecoder.h

```diff
@@ -222,9 +222,8 @@ enum GcInfoDecoderFlags
     DECODE_PROLOG_LENGTH         = 0x400,   // length of the prolog (used to avoid reporting generics context)
     DECODE_EDIT_AND_CONTINUE     = 0x800,
     DECODE_REVERSE_PINVOKE_VAR   = 0x1000,
-    DECODE_RETURN_KIND           = 0x2000,
 #if defined(TARGET_ARM) || defined(TARGET_ARM64) || defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64)
-    DECODE_HAS_TAILCALLS         = 0x4000,
+    DECODE_HAS_TAILCALLS         = 0x2000,
```

Comment:
Should these flags stay around for compat with SOS (see the comment at the top of the file)?

Reply:
Once we change the format it will be a breaking change. Diagnostics repo seems to just take a copy of the decoder once that happens.   
Otherwise, the flag is just for decoder API, it should not affect anything that does not use the new decoder. 

I think this is ok to change, but we can use the old value to be sure. This is an int32, so no shortage of bits.